### PR TITLE
new rule to make sure there's no class 'toc-sidebar' on the body

### LINKED ISSUE #315: Make sure the class 'toc-sidebar' is added by JS and JS only
The editors shouldn't add the class 'toc-sidebar' to `<body>` themselves as it's causing issues when JS is disabled on small screens (see https://github.com/w3c/tr-design/pull/91#issuecomment-208467559)

### PR DESCRIPTION
**Pull request by deniak**
fix #315

### CODE REVIEW

File: lib/l10n-en_GB.js [hunk 1]
```diff
@@ -182,6 +182,9 @@ for extended information about the publication rules.'
 ,   "style.script.not-found":   "A link to the script <code>fixup.js</code> is required (eg, <code>&lt;script src=\"//www.w3.org/scripts/TR/2016/fixup.js\"&gt;&lt;/script&gt;</code>)."
     // style/back-to-top
 ,   "style.back-to-top.not-found":  'It is recommended to include this <em>back-to-top</em> hyperlink at the end of <code>&lt;body&gt;</code>: <code>&lt;p role="navigation" id="back-to-top"&gt;&lt;a href="#toc"&gt;&lt;abbr title="Back to top"&gt;&uarr;&lt;/abbr&gt;&lt;/a&gt;&lt;/p&gt;</code>.'
+    // style/body-toc-sidebar
+,   "style.body-toc-sidebar.class-found": 'The class "toc-sidebar" should be added by fixup.js.'
+,   "style.body-toc-sidebar.selector-fail": 'Selector "body" failed.'
```
Comment (tripu) [inline lib/l10n-en_GB.js]: Maybe better to format tag names, classes, etc with HTML, as we're doing with other rules?

    The class <code>toc-sidebar</code> should be added by <code>fixup.js</code>.

and:

    Selector <code>body</code> failed.

(I think the output looks better and is easier to understand.)

File: test/docs/style/toc-sidebar.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="shrink-to-fit=NO, width=device-width, initial-scale=1.0">
+    <title>Dummy Test</title>
+  </head>
+  <body class="toc-sidebar">
```
Comment (tripu) [inline test/docs/style/toc-sidebar.html]: Shall we add that class to some other existing document, instead of creating a new one just for this?

**Comment by tripu** [comment]
LGTM, can be merged (barring the two comments inline).

**Comment by deniak** [comment]
@tripu Good point. Both comments addressed.

**Pull request merged**